docs/FAQ.md

```diff
@@ -1,52 +1,134 @@
+​
+### Are you planning to have plugins baked into the repo? Or should they be developed in separate repos?
+​
+Contributors can add open source plugins to the plugins directory in [this monorepo](https://github.com/spotify/backstage). Integrators can then configure which open source plugins are available to use in their instance of the app. Open source plugins are downloaded as npm packages published in the open source repository.
+​
+While we encourage using the open source model, we know there are cases where contributors might want to experiment internally or keep their plugins closed source. Contributors writing closed source plugins should develop them in the plugins directory in their own Backstage repository. Integrators also configure closed source plugins locally from the monorepo.
+​
+### Any plans for integrating with other repository managers, such as GitLab or Bitbucket?
+​
+We chose GitHub because it is the tool that we are most familiar with, so that will naturally lead to integrations for GitHub being developed at an early stage.
+​
+Hosting this project on GitHub does not exclude integrations with alternatives, such as GitLab or Bitbucket. We believe that in time there will be plugins that will provide functionality for these tools as well. Hopefully, contributed by the community!
+​
+Also note, implementations of Backstage can be hosted wherever you feel suits your needs best.
+​
+### Who maintains Backstage?
+​
+Spotify will maintain the open source core, but we envision different parts of the project being maintained by various companies and contributors. We also envision a large, diverse ecosystem of open source plugins, which would be maintained by their original authors/contributors or by the community.
+​
+When it comes to [deployment](https://github.com/spotify/backstage/blob/master/DEPLOYMENT.md), the system integrator (typically, the infrastructure team in your organization) maintains Backstage in your own environment.
+​
+### Does Spotify provide a managed version of Backstage?
+​
+No, this is not a service offering. We build the piece of software, and someone in your infrastructure team is responsible for [deploying](https://github.com/spotify/backstage/blob/master/DEPLOYMENT.md) and maintaining it.
+​
+### How secure is Backstage?
+​
+It depends on what you do with Backstage and how you set up your environment. The security responsibility is placed on whomever deployed that instance of the Backstage app. We provide the code, but you take care of the security measures.
```

Comment:
Is this really our approach? From a legal point of view it makes sense, but we want to make sure plugins are development with security in mind, and we want to make sure boundaries that we set up as a part of our APIs encourage best practices.